api/src/pcapi/core/educational/api/offer.py

```diff
@@ -260,6 +261,7 @@ def create_collective_offer_template(
         interventionArea=offer_data.intervention_area or [],
         priceDetail=offer_data.price_detail,
         formats=offer_data.formats,  # type: ignore[arg-type]
+        author=current_user,
```

Comment:
On avait déjà le paramètre `user`, donc tout simplement : 
```suggestion
        author=user,
```